src/Sylius/Component/User/Context/CustomerContextInterface.php

```diff
@@ -22,4 +22,9 @@
      * @return CustomerInterface|null
      */
     public function getCustomer();
+
+    /**
+     * @param CustomerInterface $subject
+     */
+    public function setCustomer(CustomerInterface $subject);
```

Comment:
I needed this when writing [specs here](https://github.com/Sylius/Sylius/pull/3230/files#diff-09bf6881a66fdabc66e869b85477d269R22), it feels weird to me too. Is it ok to use `CustomerContext` there?